Lombiq.Walkthroughs/ResourceManagementOptionsConfiguration.cs

```diff
@@ -13,11 +13,11 @@ static ResourceManagementOptionsConfiguration()
     {
         _manifest.DefineResource("$" + nameof(FeatureIds.Area), FeatureIds.Area);
 
+        // We can't set shepherd.min.css here, since the file comes from a node module and it's originally a css file,
+        // so we are not processing it. It's also somewhat minified by default.
```

Comment:
Needs line breaks but this is what the content I suggest:

```suggestion
        // We can't set a minified CSS here, since the file comes from a node module and it's originally a CSS file already, thus we are not processing it (we only do that with SCSS). It's also somewhat minified by default so that's not a big issue.
```